components/Avatar/Avatar.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import Person from '@material-ui/icons/Person';
+import Icon from '../Icon/Icon';
+import { colors } from '../shared/theme';
+
+const borderStyle = 'border-radius: 50% !important;';
```

Comment:
`!important` is it necessary?

Reply:
not really, maybe just for some edge cases

Reply:
but I removed